models/zeroex/optimism/zeroex_optimism_api_fills.sql

```diff
@@ -96,18 +96,18 @@ v4_limit_fills_no_bridge AS (
             COALESCE(zeroex_tx.affiliate_address, fills.feeRecipient) AS affiliate_address,
             (zeroex_tx.tx_hash IS NOT NULL) AS swap_flag,
             (fills.feeRecipient in 
-                ('0x9b858be6e3047d88820f439b240deac2418a2551','0x86003b044f70dac0abc80ac8957305b6370893ed','0x5bc2419a087666148bfbe1361ae6c06d240c6131')) 
+                (0x9b858be6e3047d88820f439b240deac2418a2551,0x86003b044f70dac0abc80ac8957305b6370893ed,0x5bc2419a087666148bfbe1361ae6c06d240c6131)) 
```

Comment:
I am not sure if the third address is a typo during migration:
```suggestion
                (0x9b858be6e3047d88820f439b240deac2418a2551,
                0x86003b044f70dac0abc80ac8957305b6370893ed,
                0x5bc2419a087666148bfbe1361ae6c06d240c6131)) 
```